.github/workflows/test.yml

```diff
@@ -206,7 +206,7 @@ jobs:
           TOTAL_PACKAGES: $TOTAL_PACKAGES
 
       - name: Create MRT credentials file
-        if: env.IS_NOT_FORK == 'true' && env.IS_TEMPLATE_FROM_RETAIL_REACT_APP == 'true'
+        if: env.IS_NOT_FORK == 'true' && env.IS_TEMPLATE_FROM_RETAIL_REACT_APP == 'true' && env.DEVELOP == 'true'
```

Comment:
Where did this change come from?

Reply:
Unrelated small change on GA, I thought to include it in this PR.

Right now, we're running the Create MRT credentials file step in all internal builds, and we only need to run that step when we merge into `develop`.